pkg/cmd/server/start/start_master.go

```diff
@@ -387,8 +387,11 @@ func (m *Master) Start() error {
 
 	controllersEnabled := m.controllers && m.config.Controllers != configapi.ControllersDisabled
 	if controllersEnabled {
-		// informers are shared amongst all the various controllers we build
-		informers, err := NewInformers(*m.config)
+		openshiftControllerInformers, err := NewInformers(*m.config)
+		if err != nil {
+			return err
+		}
+		kubeControllerInformers, err := NewInformers(*m.config)
```

Comment:
does this double the watches?!

Reply:
> does this double the watches?!

Yes it does.  We plan to run in a separate process shortly. Already cleared with clayton.